### ISSUE TITLE
[Feature Request] Support Multiple Training Sessions

### ISSUE THREAD

**Issue by heywhy** [aradzie/keybr.com#142]
So, I just got a new split keyboard and I'm looking to practice using the platform but I don't want to delete my previous statistics to start. I would like to create a different session that captures my training data using this new keyboard as a different typing data.

**Comment by arthurzenika** [comment]
I agree, that would be neat. Even for a dual learning experience where you can train on multiple keyboards (or layouts) and compare the stats or evolution.

**Comment by heywhy** [comment]
Exactly. It's going to be a game changer if this feature gets supported.

**Comment by xingfanxia** [comment]
Yeah me too, just got a glove80. Training sessions on different keyboards should be separate.

**Comment by mmai** [comment]
There is a similar feature request here : #10
